### PR TITLE
Feature/json

### PR DESCRIPTION
**Pull request by michal-halasa**
@jakubpetrik @thakel 
ak sa chcete pozriet, kam kracam
vzhladom na tu cyklovatost kodu, aby JSON konfig fungoval aj pre Obj-C kod, tak by sa muselo bridgovat, takze zatial to tam nie je
neviem, ci/ako to cele zatial funguje, s testami zacinam
zacinam pridavat aj podporu pre Travis

### CODE REVIEW

File: Demo/ConfigDemo/Resources/Configuration/SampleConfig.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        "One": 1
+    },
+    "SampleNumber": 42,
+    "AnArray": [
+        "One",
+        "Two",
+        "Three"
+    ],
+    "SampleURL": "https:\/\/sampleurl.eu\/abc\/"
+}
```
Comment (jakubpetrik) [inline Demo/ConfigDemo/Resources/Configuration/SampleConfig.json:13]: JSON supports also boolean values. Please add them.

File: INLConfig/INLConfigGenerator/main.swift [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+}
+
+func typeForValue(_ value: AnyObject) -> String {
+    if value is String { return "NSString" }
+    if value is NSNumber { return "NSNumber" }
+    if value is NSArray { return "NSArray" }
+    if value is NSDictionary { return "NSDictionary" }
+    if value is Data { return "NSData" }
+    assert(false, "Invalid type \(value)")
+    return "Any"
```
Comment (jakubpetrik) [inline INLConfig/INLConfigGenerator/main.swift:131]: Under Swift 3, we should probably use types without NS prefix.

File: INLConfig/INLConfigGenerator/main.swift [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+            lang = .swift
+        }
+        else if CommandLine.arguments[2] == "--objC" {
+            lang = .objC
+        }
+    }
+    
+    if 3 < CommandLine.arguments.count {
+        rootPath = CommandLine.arguments[3]
+    }
```
Comment (jakubpetrik) [inline INLConfig/INLConfigGenerator/main.swift:212]: Please don't use yoda style compare.

File: INLConfig/INLConfigGenerator/main.swift (hunk 2 above)
Comment (radimhalfar) [reply INLConfig/INLConfigGenerator/main.swift:131]: I assume yes, but I see it reasonable for Obj-C code generation ? What do you think ?

**Comment by jakubpetrik** [comment]
Closing since it is not relevant anymore.

**Pull request closed without merging**